ci/properties/cmake-multi-platform.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "CMake based multi-platform projects",
+    "description": "Build and test a CMake based project on multiple platforms.",
+    "iconName": "cmake",
```

Comment:
@andrewakim Do we know what this `iconName` field controls? If it's used to select an icon, should we consider having some sort of adorned icon to distinguish them?

Reply:
From the [GitHub docs](https://docs.github.com/en/actions/using-workflows/creating-starter-workflows-for-your-organization#creating-a-starter-workflow):

> iconName - Optional. Specifies an icon for the workflow that is displayed in the list of workflows. iconName can one of the following types:
>
>    - An SVG file that is stored in the workflow-templates directory. To reference a file, the value must be the file name without the file extension. For example, an SVG file named example-icon.svg is referenced as example-icon.
> 
>    - An icon from GitHub's set of [Octicons](https://primer.style/octicons/). To reference an octicon, the value must be octicon <icon name>. For example, octicon smiley.

Reply:
Thanks @jtamsut. I went looking to see if there were any public domain icons that could help distinguish these workflows, but I couldn't really find any. I did find that the current CMake icon we are using has been updated, but [the version available on Wikimedia Commons](https://commons.wikimedia.org/wiki/File:Cmake.svg) is only 200 x 200. I think we are fine skipping this for now.